charts/webapi/templates/deployment.yaml

```diff
@@ -120,21 +120,26 @@ spec:
             periodSeconds: 10
             successThreshold: 1
             failureThreshold: 3
-            initialDelaySeconds: 8            
-          startupProbe:
+            initialDelaySeconds: 30           
+          # startupProbe:
+          #   httpGet:
+          #     path: /health/liveness
+          #     port: http
+          #     scheme: HTTP
+          #   timeoutSeconds: 1
+          #   periodSeconds: 10
+          #   successThreshold: 1
+          #   failureThreshold: 30
+          #   initialDelaySeconds: 5
+          readinessProbe:
             httpGet:
-              path: /health/liveness
+              path: /
```

Comment:
Why is the readiness probe now at `/`? using the liveness endpoint for readiness wasn't a great approach but is this better?

Reply:
@james-hollinger Does health/readiness endpoint exist? If so, it can be used as the correct path.